test.properties

```diff
@@ -7669,22 +7686,20 @@ test.batch.run.property.query[functional-orcllinux9-tomcat90-mysql57-jdk8]=\
         modules-integration-bundle-oracle193-jdk21_zulu,\
         modules-integration-bundle-postgresql121-jdk21_zulu,\
         modules-integration-bundle-sqlserver2017-jdk21_zulu,\
-        portal-renamed-tomcat-in-path-mysql57-jdk8,\
-        portal-startup-space-in-path-mysql57-jdk8
+        portal-renamed-tomcat-in-path-mysql57-jdk21_zulu,\
+        portal-startup-space-in-path-mysql57-jdk21_zulu
 
     test.batch.names[portal-release][portal]=\
-        empty-osgi-core-dir-mysql57-jdk8,\
-        functional-bundle-tomcat-hypersonic20-jdk8,\
-        functional-bundle-tomcat-mariadb102-jdk8,\
-        functional-bundle-tomcat-mysql57-jdk11_zulu,\
-        functional-bundle-tomcat-mysql57-jdk17_open,\
+        empty-osgi-core-dir-mysql57-jdk21_zulu,\
+        functional-bundle-tomcat-hypersonic20-jdk21_zulu,\
+        functional-bundle-tomcat-mariadb102-jdk21_zulu,\
         functional-bundle-tomcat-mysql57-jdk21_zulu,\
-        functional-bundle-tomcat-mysql80-jdk8,\
-        functional-bundle-tomcat-postgresql121-jdk8,\
+        functional-bundle-tomcat-mysql80-jdk21_zulu,\
+        functional-bundle-tomcat-postgresql121-jdk21_zulu,\
         functional-bundle-ubuntu18-tomcat90-mysql57-jdk8,\
-        functional-smoke-bundle-wildfly180-mysql57-jdk8,\
-        functional-smoke-bundle-wildfly230-mysql57-jdk8,\
-        functional-smoke-bundle-wildfly261-mysql57-jdk8,\
+        functional-smoke-bundle-wildfly180-mysql57-jdk21_zulu,\
+        functional-smoke-bundle-wildfly230-mysql57-jdk21_zulu,\
+        functional-smoke-bundle-wildfly261-mysql57-jdk21_zulu,\
```

Comment:
these didn't show in job summary

Reply:
@vicnate5 in [old release test job summary](https://ffa45b50ebdf44555fdc85cdffd74f207ce435b75a272acb63d3f66-apidata.googleusercontent.com/download/storage/v1/b/testray-results/o/2024-07%2Ftest-1-1%2Ftest-portal-release%2F4450%2Fjob-summary%2Findex.html.gz?jk=AYBlUPBdAE_OhsnalfL_cFbh7EilBTE5f_rbYtPBJV0Qnz7idYXuEo8qcG66gJeXJFm8RMzcCOrBXCXz_73fBcCZuTDKN70Svcpc_pt2JTO1lwqLD2yEgvaSYQ_JZ4KBAh390kya7lcGo_PENW9ySwM2cLOz7GXQZF1kQu0FR3PdrYld4OXDKPqrId_jqsMqAb5QagZALUSaQ2dAGr2T71BUKqydU0fxegPGHVGCV-Lnzb8ZUpOaEuRhHPgIlbYn2-cFWqNUcLTZoIf7f2hLq5O50w4Yf-r9Xf3_OTjrIUsYV93G-lwJYLlLeV4UzIXi-Tn9W96hNOyztuhbiPSeEqzQQrorAEPJFVpY1ReF4Ic4qXTUBIrQQUDQ8zktgyHHhEmp5uKpleDza-wypXJ0JFKHKcoZcbrR7b4XcudqFSCv9yHt_eWLMUccHrvCnE_9aI8E78_t8Tr7qSTBzNhm5b2mjmkO-fmr1s90I2Z5PhZhaNfwwa-DbolINL8ZyiQSk-6RqcpHo1sTapkv_TUsa9Kb9ZfRrtzc4SOVSNlvhSKbvP7V4iqZqsjbxSgnyO5K1Tv8QrcezKiJcoZk2zDWgxKdbaQlBIBlghaFod0fOlfr6uEaIrHg80qOXTp1TiyHJ5KRNfvKoDrhdcA-DgYnkgjYa0rjcPiJ21-QyCuxhIjXikEjj1cpBBIMQE0BKdGChuVryZOTihJ3FZMQ_l32nNvMVwtS_fwN2hJySSzVKsFUD_mxeUv_vdCC28MeFzMTTpbG6jsa1Bf69fXGtyIU8IHflQ5zkJP-_jnvU6vES6DiQjaxwXUSikCz0DiuqOChqCcM2gkVU1FrKsAL-ncQr1k8_3HPSmq-2hwHq6utUxhGSh46x90S4U94tmwDcdH8QhsbUYnxBeq-tzsdRhqUOjo-LwCcUvppv_6zERYWhvG9_uEXAw5NoyW6z2Kkzl1sy62TGanBQDgMFd2itOX0Vmyl_jFEVML4qOQXLW1krrGg1NIc9EY25ZSSGLmJDRBRx7-Vh77Nxubui2trS83cbJPCzLC_6I6FUuYQPHp7OTPHeP0pho9o94wAek4iYxtNxPZ8Jc6yKkdEd81NFb1qfx1HV4_TfM3ixMEbcatrxn8dMckPcSgEru9j-Nc4Iq6iz5g7PIxFJu2AcLoEndbb5yruxyRHLLnmdV-q4tzleEnW_mPkSXUF2tizJsHVS00-D2zoq_FCRBh_JScL6KWQldnj7E42rWPz5nsN7pvkKvRL3V2bRw&isca=1)
I don't see these smoke wildfly test batches run either. Maybe no test match the query criteria?

Reply:
Gotta check the query.

We also should remove wildfly 18 and 23, but that should be a new ticket at this point.

Reply:
@julschong this can be covered by https://liferay.atlassian.net/browse/LPD-31337